devbot/database.py

```diff
@@ -51,4 +60,15 @@ class XKCD(SQLAlchemyBase):
     year = sa.Column(sa.String(4))
 
 
+class Task(SQLAlchemyBase):
+    __tablename__ = "task"
+    # Representation of a task to be delayed
+    id = sa.Column(sa.Integer(), primary_key=True)
+    datetime = sa.Column(sa.DateTime())
+    channel = sa.Column(sa.String(64))  # Stores the channel-id of the output channel
```

Comment:
Currently a channel id is 18 characters long. We can probably get away with a max length of 18 or 19 characters for a long time. Database optimisation, yo.